chirripo/chirripo
# Enables usage of PHP_MEMORY_LIMIT  and PHP_MAX_EXECUTION_TIME env variables

## Issue Reference

Issue Number: #47 

## What does this Pull Request do?

Maps the PHP_MEMORY_LIMIT and PHP_MAX_EXECUTION_TIME env variables to the env variables in the .env file to allow to the users to change this values depending on their requirements.

## How can this Pull Request be tested?

Build a new project and set custom values to that variables into the .env file and once the env is up, go and check your php memory limit and max execution time , they should have the values you just set into the the .env


## Any additional notes/comments?
